<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarheadMainClass</class>
 <widget class="QMainWindow" name="WarheadMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WarheadMain</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open_from_file"/>
   <addaction name="action_save"/>
   <addaction name="action_save_as_file"/>
   <addaction name="action_metal_material"/>
   <addaction name="action_charge_material"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_new">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="action_new_from_templ">
   <property name="text">
    <string>从模板新建</string>
   </property>
   <property name="toolTip">
    <string>从模板新建</string>
   </property>
  </action>
  <action name="action_save_as_file">
   <property name="text">
    <string>另存</string>
   </property>
   <property name="toolTip">
    <string>另存</string>
   </property>
  </action>
  <action name="action_save_db">
   <property name="text">
    <string>保存/另存到数据库</string>
   </property>
   <property name="toolTip">
    <string>保存/另存到数据库</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="action_save_templ">
   <property name="text">
    <string>保存为模板</string>
   </property>
   <property name="toolTip">
    <string>保存为模板</string>
   </property>
  </action>
  <action name="action_open_from_file">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="action_open_from_db">
   <property name="text">
    <string>从数据库打开</string>
   </property>
   <property name="toolTip">
    <string>从数据库打开</string>
   </property>
  </action>
  <action name="action_material_lib">
   <property name="text">
    <string>材料库</string>
   </property>
   <property name="toolTip">
    <string>材料库</string>
   </property>
  </action>
  <action name="action_operator_lib">
   <property name="text">
    <string>算子库</string>
   </property>
   <property name="toolTip">
    <string>算子库</string>
   </property>
  </action>
  <action name="action_unit">
   <property name="text">
    <string>系统单位制</string>
   </property>
   <property name="toolTip">
    <string>系统单位制</string>
   </property>
  </action>
  <action name="action_metal_material">
   <property name="text">
    <string>金属材料</string>
   </property>
   <property name="toolTip">
    <string>金属材料</string>
   </property>
  </action>
  <action name="action_charge_material">
   <property name="text">
    <string>装药材料</string>
   </property>
   <property name="toolTip">
    <string>装药材料</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="warheadmain.qrc"/>
 </resources>
 <connections/>
</ui>
